Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -113,15 +113,22 @@ define([
      *                  be treated as if their alpha is 1.0 everywhere.  When this property is false, memory usage
      *                  and texture upload time are potentially reduced.
      * @param {GetFeatureInfoFormat[]} [options.getFeatureInfoFormats] The formats in which to get feature information at a
-     *                                 specific location when {@see UrlTemplateImageryProvider#pickFeatures} is invoked.  If this
+     *                                 specific location when {@link UrlTemplateImageryProvider#pickFeatures} is invoked.  If this
      *                                 parameter is not specified, feature picking is disabled.
-     *
+     * @param {Boolean} [options.deferReadiness=false] True to defer full construction of this instance until a later call to {@link UrlTemplateImageryProvider#reinitialize}.
```

Comment:
As a side note, if we assume that every time we construct an `UrlTemplateImageryProvider` instance with `deferReadiness : true` it's because the `readyPromise` is not immediately resolved, I can remove the `hasReadyPromise` property. My experience is limited to only two imagery providers, so I didn't want to assume these were mutually inclusive.